### DISCUSSION TITLE
Inpaint masking window not adjusting

### DISCUSSION THREAD

**Discussion by vivekpayasi** [mashb1t/Fooocus#44] [Q&A] [answered]
I have an image of size 1024*1024. When I drag this image to inpaint window, the window adjusts to show the full image and icons for brush, undo etc. become smaller. I'm trying to generate mask using SAM. Now the mask is getting generated properly but the window in which the mask is shown (ss below) is not adjusting to size of the image (similar to the inpaint window)

<img width="847" alt="image" src="https://github.com/mashb1t/Fooocus/assets/9135530/42d2c95a-0a00-4bb8-8538-c96388bff348">

Now that prohibits me from adding/drawing over the mask. For example, if I want to mask the arm on the right as well, I'm not able to do that.

I tried creating an issue, but it is showing a link to Discussion of the original Fooocus. 

Can someone help me in this regard? 

I'm using Colab notebook to run this.

**Comment by mashb1t** [comment]
This is a weird Gradio issue I've also already encountered in the past. Try unchecking/checking the Advanced Masking Features checkbox to close/open the mask upload. I'll check what i can do to redraw the output element.

**Reply by mashb1t** [reply, marked as answer]
@vivekpayasi fixed in main, see https://github.com/mashb1t/Fooocus/commit/6f479a5040c6a1207a501aa14cadb178b00e6df8

**Reply by vivekpayasi** [reply]
Hey @mashb1t , thank you very much for fixing it so quickly. Let me test it and get back to you.

**Reply by vivekpayasi** [reply]
It's working perfectly. Thanks once again.
